src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -76,24 +75,19 @@ public void worksFine() throws Exception {
         );
     }
 
-    // @todo #78:30min With #78 solved, HtWire stopped closing the connection
-    //  when send() is called and the following test stopped working for no
-    //  clear reason. Investigate the cause of this, fix it and unignore this.
-    @Ignore("see todo above")
+    // @todo #87:30m For now I can't find proper OOP alternative for waiting
+    //  (such in this case). Needs to create a new one.
     // @checkstyle MagicNumberCheck (1 line)
     @Test(expected = TimeoutException.class, timeout = 1000)
     public void failsAfterTimeout() throws Exception {
         // @checkstyle MagicNumberCheck (1 line)
         final long timeout = 100;
-        try (Socket blocker = new Socket()) {
-            blocker.connect(this.server.getLocalSocketAddress());
-            new HtTimedWire(
-                new HtWire(
-                    this.server.getInetAddress().getHostAddress(),
-                    this.server.getLocalPort()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
-        }
+        new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(timeout + timeout);
+                return input;
+            },
+            timeout
```

Comment:
@aivinog1 It's not clear whether timeout actually happens after 100 ms as specified here, or it waits full 100 + 100 ms until the function finishes execution and throws the `TimeoutException` only then. I think we need another test for that.

Reply:
Good point, thank you! I'll add another test.

Reply:
Done.